orgs/terraform-opentelekomcloud-modules/people/members.yml

```diff
@@ -0,0 +1,6 @@
+---
+users:
+  - name: "Rodion Gyrbu"
+    login: "lego963"
+    visibility: Public
```

Comment:
Why there is only one member?

Reply:
you want to be added?

Reply:
Currently only Rodion included

Reply:
Currently there are four members:
![image](https://user-images.githubusercontent.com/8591561/120800096-3a380880-c548-11eb-9107-eb7f883da0f5.png)

Reply:
okay